### PR TITLE
Pkiambo's update

### PR DESCRIPTION
**Pull request by pkiambo**

### CODE REVIEW

**Comment by ltagliaferri** [comment]
Thank you for opening this pull request, but it does not follow the [contributing guidelines](https://github.com/do-community/cloud_haiku#contributing) that are outlined in this repo, so we are closing this PR. If you would like to contribute in the future, please review this guidelines before opening a new pull request.

**Comment by pkiambo** [comment]
Can I reattempt the PR?

**Comment by beardicus** [comment]
We are happy to review any new pull requests, or reopen old requests that have been updated. You can push right to this PR branch to update it, then let us know when we should take a look. Or you could open a new pull request if you'd like. Thanks!

File: pkiambo.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Hi My name is Patsy.
```
Comment (pkiambo) [inline pkiambo.txt]: --- 
layout: haiku 
title: Mau Mau 
author: Patsy
---

I’m mau mau daughter<br>
For land and self governance<br>
The uprise lives on<br>

**Comment by pkiambo** [comment]
How do I reopen the closed PR?

**Comment by beardicus** [comment]
You can't reopen this PR, but this repo's owners can (myself included). If you push a valid contribution to your `pkiambo:pkiambo-haiku` branch we can reopen and test and merge.

**Comment by pkiambo** [comment]
I would appreciate a reopen.

Thanks

**Comment by pkiambo** [comment]
> You can't reopen this PR, but this repo's owners can (myself included). If you push a valid contribution to your `pkiambo:pkiambo-haiku` branch we can reopen and test and merge.

an updated push with valid contribution has been submitted.
To https://github.com/pkiambo/cloud_haiku
   bcc60d0..4194e59  pkiambo-haiku -> pkiambo-haiku

**Comment by pkiambo** [comment]
> > You can't reopen this PR, but this repo's owners can (myself included). If you push a valid contribution to your `pkiambo:pkiambo-haiku` branch we can reopen and test and merge.
> 
> an updated push with valid contribution has been submitted.
> To https://github.com/pkiambo/cloud_haiku
> [bcc60d0](https://github.com/do-community/cloud_haiku/commit/bcc60d0eee7fb19203e48c8de9187714068fd40e)..[4194e59](https://github.com/do-community/cloud_haiku/commit/4194e59fdd70989abca84f968d78ed7253291616) pkiambo-haiku -> pkiambo-haiku

It has been corrected:

Pkiambo haiku #398
This is an updated PR To https://github.com/pkiambo/cloud_haiku bcc60d0..4194e59
Screen Shot 2019-11-07 at 5 06 04 PM

**Comment by beardicus** [comment]
removing invalid tag here re: #398

**Pull request closed without merging**